app.rb

```diff
@@ -0,0 +1,134 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class App
+  attr_accessor :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @rentals = []
+    @people = []
+  end
+
+  def list_books
+    @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+  end
+
+  def list_books_with_index
+    @books.each_with_index { |book, i| puts "#{i}) Title: \"#{book.title}\", Author: #{book.author}" }
+  end
+
+  def list_people
+    @people.each do |person|
+      puts "[#{person.class.name}] Name: \"#{person.name}\", ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def list_people_with_index
+    @people.each_with_index do |person, i|
+      puts "#{i}) [#{person.class.name}] Name: \"#{person.name}\", ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    student_or_teacher = gets.chomp.to_i
+    if student_or_teacher == 1
+      create_student
+    elsif student_or_teacher == 2
+      create_teacher
```

Comment:
- [ ] Please make sure that the app works properly. Currently, the app produces an error when the user tries to create a person (student or teacher). The following error prevented me from creating rentals, so please make sure that this feature is also working as intended after fixing this issue.

![image](https://github.com/Errytagedesign/schoolLibrary/assets/62290545/83698ded-3e1b-43b1-b85f-0d883388130e)